### PR TITLE
introducing newsletter

### PR DESCRIPTION
**Pull request by julien51**
# Description

Blog post to introduce the newsletter!

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/blog/introducing-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+authorName: Julien Genestoux
+publishDate: December 12, 2019
+description: Unlock lets you easily collect email addresses from members to create your own paid-for mailing list, using Ethereum and Non Fungible Tokens
+---
+
+There are many kinds of memberships. Going back to the origins of the internet, newsletters have always been a popular style of membership. Today, we're excited to show you how you can easily use a lock to create your own newsletter membership with Unlock!
+
+# The Defiant
+
+A couple of weeks ago, while at DevCon in Osaka, I was introduced to [Camila](https://twitter.com/camirusso) who publishes one of the most insightful newsletters about the DeFi movement. I told her about Unlock and the fact that we [tokenize memberships](/tokenizing-memberships/), we quickly agreed that she needed to provide the ability to join her mailing list using crypto.
```
Comment (cnasc) [inline unlock-protocol.com/blog/introducing-newsletter.md]:
```suggestion
A couple of weeks ago, while at DevCon in Osaka, I was introduced to [Camila](https://twitter.com/camirusso) who publishes one of the most insightful newsletters about the DeFi movement. When I told her about Unlock and the fact that we [tokenize memberships](/tokenizing-memberships/), we quickly agreed that she needed to provide the ability to join her mailing list using crypto.
```

File: unlock-protocol.com/blog/introducing-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Today, that's possible! you can [become a member to The Defiant](https://bit.ly/the-defiant). If you go fast, you can join the [OG membership](https://etherscan.io/address/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf) (limited to 100 members) for 70 DAI, or join her [regular membership](https://etherscan.io/address/0x43154efc9cb33c80833c0dec1e15bb9cfc1275e5) for the next year, at 100 DAI!
```
Comment (cnasc) [inline unlock-protocol.com/blog/introducing-newsletter.md]:
```suggestion
Today, that's possible! You can [become a member of The Defiant](https://bit.ly/the-defiant) using DAI. If you join quickly, you can become an [OG member](https://etherscan.io/address/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf) (limited to 100 members) for 70 DAI. If you miss the OG perk, you can still join her [regular membership](https://etherscan.io/address/0x43154efc9cb33c80833c0dec1e15bb9cfc1275e5) for the next year, at 100 DAI!
```

File: unlock-protocol.com/blog/introducing-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Of course, like any other key, The Defiant membership keys are Non Fungible Tokens and your favorite crypto wallet should show them. Here's for example, my [OG membership on OpenSea](https://opensea.io/assets/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf/1).
```
Comment (cnasc) [inline unlock-protocol.com/blog/introducing-newsletter.md]:
```suggestion
Of course, like any other key, The Defiant membership keys are Non Fungible Tokens and your favorite crypto wallet should show them. For example, here's my [OG membership on OpenSea](https://opensea.io/assets/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf/1).
```

File: unlock-protocol.com/blog/introducing-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+# How does it work
```
Comment (cnasc) [inline unlock-protocol.com/blog/introducing-newsletter.md]:
```suggestion
# How It Works
```

File: unlock-protocol.com/blog/introducing-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+First, the creator has deployed their own lock on the Ethereum chain. Once you are on the newsletter landing page, you can enter your email address. As you submit, the address is saved inside the metadata of your token.
```
Comment (cnasc) [inline unlock-protocol.com/blog/introducing-newsletter.md]:
```suggestion
First, the creator deploys their own lock on the Ethereum chain. Once you are on the newsletter landing page, you can enter your email address. As you submit, the address is saved inside the metadata of your token.
```

**Pull request merged**